### ISSUE TITLE
github issue tracker workflow update

### ISSUE THREAD

**Issue by lkrocek** [dalaranwow/dalaran-wow#5589]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Many tickets are closed, but they have some labeled as waiting etc.

**Expected behaviour**: Tell us what should happen instead
Usually is workflow to make a ticket "deployed on live", "won't fix", "duplicate" etc. to be closed. I discovered there is so many tickets which aren't done and marked as closed(means nothing to do with it).

**Steps to reproduce the problem**:

1. https://github.com/dalaranwow/dalaran-wow/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Fixed+-+Waiting+for+PTR+update%22+
2. https://github.com/dalaranwow/dalaran-wow/issues?utf8=%E2%9C%93&q=label%3A%22Fixed+-+Ready+to+deploy.%22+

This is so epic fail of this issue tracker, because these **tickets will be ignoring forever!**

**Comment by lkrocek** [comment]
you can check example here: https://github.com/dalaranwow/dalaran-wow/issues/870

and announced as fixed here: http://www.dalaran-wow.com/changelog/21/the-path-to-a-new-dawn which is **not true**

**Comment by lkrocek** [comment]
It seems @Lothloryen and @Rushor live in different dimensions so I kindly ask you please review all not finished labeled tickets (and ignore closed status)!

**Comment by Chenocide** [comment]
I'm not sure if you were around back then, but this is the changelog that Loth decided to post "in advance" so that people would know what would be fixed. Since Loth can't multitask and since he's working on Ruby Sanctum, about half of the changelog never made it to live. The correction here is that Rushor can't push to the live realm, so this is mostly a matter of Loth finishing whatever it is that he is spending his time on. 

Here's the announcement:
http://www.dalaran-wow.com/news/124/changelog-the-path-to-a-new-dawn/

"This work is still being deployed bit by bit, every few days with a short maintenance. But as it was taking longer than we expected to publish the Changelog, we decided to let you know what is already done.

As always, since the list is vast, if something is missing after some days, feel free to point it out for us in the respective issue on the bugtracker with a comment, and it will be scheduled to be deployed in the next maintenance."

I agree with you, this is not how you run an issue tracker and this is not how you do a changelog but I personally believe that we'll see a difference after Ruby Sanctum.

**Comment by lkrocek** [comment]
Thanks for response @Chenocide, while Ruby Sanctum should be live next week I can pray for it :)

PS: even this you guys are doing good things! thanks.

**Comment by lkrocek** [comment]
you didn't understand this, @Golganeth problem is changelog inconsistency, changelog should be saying what changed from last version(update) and the date shown there is 4 month old and most of this items will be not fixed before next patch with ruby sanctum, so probably should be roadmap to be marked as to be fixed in future not connected with current time, ahh doesn't matter just is really confusing...

**Comment by Golganeth** [comment]
> 4. probably loth will check closed tickets for last time

"Probably" that is wrong. We check the fixes, what is already tested and what is ready to be applied to the live realm and that's what goes to the changelog. Closing the issue and adding a label to it is just a way of having stuff "sorted out" on the tracker. Things such as testing and changelog drafts are handled internally. 

If there is something we missed to apply (a closed issue without the PTR label, for example) please feel free to point it out on the respective issue. Also not the place to discuss (forums would be better) so please keep this place clean!
